.travis.yml

```diff
@@ -4,6 +4,7 @@ env:
   - NODE_VERSION="v0.10"
   - NODE_VERSION="v0.12"
   - NODE_VERSION="iojs-v1.0"
+  - NODE_VERSION="iojs-v2.0"
```

Comment:
BTW `iojs-v1` and `iojs-v2` is a better version spec, as it tracks the latest minor+patch version in the given major version.